from_cpython/setup.py

```diff
@@ -743,6 +743,10 @@ def detect_modules(self):
             missing.extend(['imageop'])
 
         # readline
+        # Pyston change: in some container system, the dependencies are provided by
```

Comment:
My vote is to keep it as close to the way that CPython does it as possible.  If it's something in their configure script, then it seems like we should put it in our CMake configuration.

I don't know if it matters here, but I think there's also a small confusion between `CPPFLAGS` and `CXXFLAGS` -- confusingly, `CPPFLAGS` doesn't have anything to do with C++, and is instead about the "C pre-processor".  I wouldn't be surprised if our cmake configuration just doesn't pick up that variable when it should.